lib/middle_ai/tracer.ex

```diff
@@ -5,9 +5,17 @@ defmodule MiddleAi.Tracer do
 
       @tracer_ref Keyword.fetch!(opts, :tracer_ref)
 
-      @spec start_trace(String.t(), String.t(), map(), String.t(), String.t(), String.t()) ::
+      @spec start_trace(
+              String.t(),
+              String.t(),
+              map(),
+              String.t(),
+              String.t(),
+              String.t(),
+              String.t()
+            ) ::
```

Comment:
maybe label the spec args? 
```suggestion
              name :: String.t(),
              model :: String.t(),
              model_params :: map(),
              user :: String.t(),
              prompt :: String.t(),
              thread_id :: String.t(),
              initial_prompt :: String.t()
            ) ::
```